ros_buildfarm/jenkins.py

```diff
@@ -100,6 +100,8 @@ def configure_view(
         template_name='generic_view.xml.em', dry_run=False):
     view_config = get_view_config(
         template_name, view_name, include_regex=include_regex)
+    if not jenkins:
```

Comment:
Ok, it's surprising to have it return either a `view_config`, `None`, or a `view`